src/components/pricing/select-plan-new/index.tsx

```diff
@@ -202,12 +212,12 @@ const SelectPlanNew: React.FunctionComponent<SelectPlanProps> = ({
     pricesForInterval(currentInterval),
   )
 
-  const forTeams: boolean = currentQuantity >= 5
+  const forTeams: boolean = currentQuantity > 1
```

Comment:
the way the new system works it's all the same, no real difference for teams outside of the number of people that can access the account, it's not a seperate product.